tests/reader_writer_100hz.test

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<launch>
+  <!-- Start Kvaser Can Bridge with Read/Write Ticks Enabled -->
+  <node name="kvaser_can_bridge" pkg="kvaser_interface" type="kvaser_can_bridge" ns="reader">
+    <param name="can_hardware_id" value="1" />
+    <param name="can_circuit_id" value="0" />
+    <param name="can_bit_rate" value="500000" />
+    <param name="enable_debug_ticks" value="true" />
+  </node>
+
+  <!-- Start Kvaser Can Bridge with Read/Write Ticks Enabled -->
+  <node name="kvaser_can_bridge" pkg="kvaser_interface" type="kvaser_can_bridge" ns="writer">
+    <param name="can_hardware_id" value="1" />
+    <param name="can_circuit_id" value="1" />
+    <param name="can_bit_rate" value="500000" />
+    <param name="enable_debug_ticks" value="true" />
+  </node>
+
+  <node name="topic_pub" type="rostopic" pkg="rostopic"
+    args="pub -r 100 /writer/can_rx can_msgs/Frame '{header: {seq: 0, stamp: {secs: 0, nsecs: 0}, frame_id: ''}, id: 546, is_rtr: false, is_extended: false, is_error: false, dlc: 8, data: [0,0,0,0,0,0,0,0]}'" />
```

Comment:
Where does the 546 value come from?

Reply:
It's 0x222 in decimal. I just picked a random number < 0x7FF. Maybe it should have some meaning behind it (like 0x555 / 1365 / 0b10101010101 so it's easy to see transposed/modified bits) but I didn't put that much thought into it - I just wanted to make the test work.